blocks/library/freeform/editor.scss

```diff
@@ -146,6 +146,10 @@
 	display: block;
 }
 
+.freeform-toolbar.has-advanced-toolbar {
+	margin-top: -85px;	/* Pull upwards the classic text toolbar when enabled */
```

Comment:
Is it always 85 pixels? I guess the advanced toolbar could have several rows?

Reply:
I tried installing TinyMCE advanced, and I see two issues, when activating the plugin, I see the "menu" toolbar pushing the block's toolbar a bit. And when adding an extra row in the settings, It's not showing up as expected too.

<img width="661" alt="screen shot 2017-09-29 at 09 55 11" src="https://user-images.githubusercontent.com/272444/31008516-537e2bd2-a4fc-11e7-815c-c459b2cb6014.png">

Reply:
Solid observation, and good point. I don't know what we can do to fix this — I know it seems hacky, but unless we have some way to calculate the height of the fully expanded toolbar, I can't think of a way to "work perfectly" in every toolbar situation. Especially since responsiveness is going to introduce still more issues. 

That said, the above seems to work fairly well considering you have _three_ toolbars. All content is editable, all content is visible, and there is room for the toolbars. Since one key philosophy is that _unselected block is preview_ and _selected block/edit mode can show more options_, although not optimal this doesn't fall outside of the principles. 

Since Gutenberg comes with up to two toolbars by default, I _feel_ like this is a decent compromise, especially since three stacked toolbars is an editor design I don't think is very good in the first place, but there for back compat purposes.

What do you think, @karmatosed?

Reply:
yep, good points, I'm ok with this tradeoff